### PR TITLE
7747 - consume the clumps api to populate the global nav

### PR DESCRIPTION
**Pull request by jongilbraith**
Pretty much what it says on the tin.

One thing to mention is that we're not doing anything with the clump link style attribute.

Once we've got more specific specs on that we can add the appropriate logic to customise the view, as well as lock down the permitted values in the CMS too

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1635)

### CODE REVIEW

File: lib/core/repository/clumps/cms.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module Core::Repository
+  module Clumps
+    class CMS < Core::Repository::Base
+      def initialize(options = {})
+        self.connection = Core::Registry::Connection[:cms]
+      end
+
+      def all
+        clumps_url = '/%{locale}/clumps.json' % { locale: I18n.locale }
```
Comment (jongilbraith) [inline lib/core/repository/clumps/cms.rb]: @tomas-stefano yes, we had this discussion the other day.

I was planning on coming back to it as a separate task to tidy these up as the CMS also needs changing (this one doesn't work under both, like some of the api end points).

Comment (tomas-stefano) [inline lib/core/repository/clumps/cms.rb]: Ok. Sounds good 👍

**Comment by tomas-stefano** [comment]
👍 👍

**Comment by jriga** [comment]
👍

**Comment by jongilbraith** [comment]
@tomas-stefano @jriga thanks both

**Pull request merged**